server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAnalysesList.js

```diff
@@ -33,49 +34,63 @@ type Props = {
   canAdmin: boolean,
   changeEvent: (event: string, name: string) => Promise<*>,
   deleteAnalysis: (analysis: string) => Promise<*>,
-  deleteEvent: (analysis: string, event: string) => Promise<*>
+  deleteEvent: (analysis: string, event: string) => Promise<*>,
+  fetchMoreActivity: () => void,
+  paging?: Paging
 };
 
 export default function ProjectActivityAnalysesList(props: Props) {
   if (props.analyses.length === 0) {
-    return <div className="note">{translate('no_results')}</div>;
+    return (
+      <div className="layout-page-side-outer project-activity-page-side-outer">
+        <div className="boxed-group boxed-group-inner">
+          <div className="note">{translate('no_results')}</div>
+        </div>
+      </div>
+    );
   }
 
   const firstAnalysis = props.analyses[0];
-
   const byDay = groupBy(props.analyses, analysis => moment(analysis.date).startOf('day').valueOf());
-
   return (
-    <div className="boxed-group boxed-group-inner">
-      <ul className="project-activity-days-list">
-        {Object.keys(byDay).map(day => (
-          <li
-            key={day}
-            className="project-activity-day"
-            data-day={moment(Number(day)).format('YYYY-MM-DD')}>
-            <div className="project-activity-date">
-              <FormattedDate date={Number(day)} format="LL" />
-            </div>
+    <div className="layout-page-side-outer project-activity-page-side-outer">
```

Comment:
Put a border around this element, and (somehow) remove from the `boxed-group`. Otherwise the right border is displayed inside the scrolling area, and you never see both top and bottom border at the same time.